browser/scripts/player.js

```
@@ -247,6 +247,10 @@ function CreatePlayer(cb) {
 	E2.core = new Core()
 	
 	E2.dom.webgl_canvas = $('#webgl-canvas')
+	if (E2.dom.webgl_canvas.length < 1)
```

should we throw an error or atleast a console.warn() if this happens?